merc_module/mercmodule.py

```diff
@@ -295,17 +372,20 @@ def MakeMoon(whichdir, whichtime):
         # B or D => smaller mass, H or L => larger (j)
         # 1 => smaller axis, 2 => larger axis
         # B or H  => small density, D or L => large density
-        FakeFile = open('FakeMoons.txt', 'r')
+
+        # @TODO: Needs to be made an absolute path?
+        FakeFile = open("FakeMoons.txt", "r")
         Fake = FakeFile.readlines()
         FakeFile.close()
-        if whichdir[0] == 'B' or whichdir[0] == 'H':
+        # @TODO: This is always going to be "B" right?  Should we simplify this logic?
```

Comment:
Same as above, we don't really need this function and I am not sure what the intention with this section was... It will also just fail if using a directory name that starts with any letter other than B, H, D, or L (which I have been in testing). But unclear what the different settings were for or why to pick one over another.